cmd/kubeadm/app/util/pkiutil/pki_helpers.go

```diff
@@ -427,8 +431,13 @@ func getAltNames(cfg *kubeadmapi.InitConfiguration, certName string) (*certutil.
 
 	// create AltNames with defaults DNSNames/IPs
 	altNames := &certutil.AltNames{
-		DNSNames: []string{cfg.NodeRegistration.Name, "localhost"},
```

Comment:
"localhost" lost?

Reply:
Thanks for pointing out the error, leave it as it is.